### ISSUE TITLE
Does ESLint not support @inheritdoc?

### ISSUE THREAD

**Issue by KnightYoshi** [eslint/eslint#11661]
I have a base class that defines the @param types, names, and descriptions. I extend the base class that overrides one of the methods of the base class. However, when using `@inheritdoc` ESLint doesn't provide any information of the object from the parameter that I'm referencing.

_foo.js_
```
class Foo {
    
    /**
     * 
     * @param {Entity} entity -
     * @param {string} attribute -
     */
    something(entity, attribute) {
         // do stuff with the arguments, 
    }
}
```

_bar.js_
```
class Bar extends Foo {
    
    /**
     * 
     * @inheritdoc
     */
    something(entity, attribute) {
         // do other stuff with the arguments, 
    }
}
```

I would expect auto-complete and hints for the `Entity` type in `Bar#something` since it's defined in `Foo#something`. However, unless I copy the same comment block into _bar.js_ I don't get any hinting for the `entity` param type like I do in `Foo#something`

**Comment by not-an-aardvark** [comment]
ESLint doesn't do anything related to autocompletion. I think you should report an issue on your editor integration instead.

**Comment by KnightYoshi** [comment]
That's fair. I wasn't sure, since I have the editor using ESLint. So I wasn't sure if it's because ESLint wasn't properly returning information about the `@inheritdoc`. I'll see what the people over at VSCode say (maybe they'll punt back over to here? LOL)
